python/packagescanner.go

```diff
@@ -114,6 +114,7 @@ func (ps *Scanner) Scan(ctx context.Context, layer *claircore.Layer) ([]*clairco
 			Name:              strings.ToLower(hdr.Get("Name")),
 			Version:           v.String(),
 			PackageDB:         "python:" + pkgDB,
+			File:              n,
```

Comment:
Do we only need this for Python? I imagine other language, like Java for sure. Probably not necessary for non-languages, right?

Reply:
Yes we'll probably want to add it for all languages, I was between adding the python part and just doing to individual language support in separate PRs, I chose to add it as a blueprint.